boards/nucleo-l452/Makefile.features

```diff
@@ -0,0 +1,15 @@
+# Put defined MCU peripherals here (in alphabetical order)
+FEATURES_PROVIDED += periph_gpio
+FEATURES_PROVIDED += periph_pwm
+FEATURES_PROVIDED += periph_rtc
```

Comment:
I think you could also add a RTT on this board, see the nucleo-l476 as an example.

Reply:
I don't think  RTT works for L4, at least running tests/periph_rtt doesn't work on L476, doesn't work in master an doesn't work at least up to realease 2017.04. Im trying to look in to this, but it got broken along the way another stm32lx broken feature.... Considering this I would rather not implement this right now, merge the board and then add RTT in the future, what do you think @aabadie ?

Reply:
rtt added